addons/web/controllers/main.py

```diff
@@ -1611,14 +1613,15 @@ def report_barcode(self, type, value, width=600, height=100, humanreadable=0):
         return request.make_response(barcode, headers=[('Content-Type', 'image/png')])
 
     @http.route(['/report/download'], type='http', auth="user")
-    def report_download(self, data, token):
+    def report_download(self, data, token, context=None):
```

Comment:
this part of the patch should not be needed anymore, see https://github.com/odoo/odoo/pull/32341/commits/2cc360b61889e4c2831c815e1f4358bcd75ccff3#diff-2c418cb78b7c562a79e7032645a537d2R876